.github/workflows/e2e-windows.yml

```diff
@@ -137,3 +135,15 @@ jobs:
         shell: pwsh
         run: |
           .\constellation.exe iam destroy --debug -y
+
+      - name: Notify about failure
+        if: |
+          failure() && github.event_name == 'workflow_call'
+        continue-on-error: true
+        uses: ./.github/actions/notify_failure
```

Comment:
Please trigger a failure in a dispatch run while relaxing the if statement to check that the notification actually works.